config/manifests/bases/operator-certification-operator.clusterserviceversion.yaml

```diff
@@ -37,15 +65,17 @@ spec:
   keywords:
   - certification
   - pipeline
+  - openshift
   links:
-  - name: Operator Certification Operator
-    url: https://github.com/redhat-openshift-ecosystem/operator-certification-operator
+  - name: Operator Certification CI Pipeline
+    url: https://github.com/redhat-openshift-ecosystem/certification-releases/blob/main/4.9/ga/ci-pipeline.md
+  - name: Red Hat OpenShift Pipelines
+    url: https://github.com/openshift/tektoncd-operator
   maintainers:
-  - email: jomckenz@redhat.com
-    name: jomkz
+  - email: certification-operator-dev-team@redhat.com
```

Comment:
pending - need to update this attribute with the actual email @jomkz created.

Reply:
@itroyano I am anticipating that the request for the email alias will be approved and the email we will use will be:

`certification-operator-devel@redhat.com`

Reply:
updated